bin/copyConcordanceCheckData.sh

```diff
@@ -213,7 +220,7 @@ else
 		# Process this sample sheet / run.
 		#
 		filePrefix="$(basename "${sampleSheet%%.*}")"
```

Comment:
This will only work when there are only dots in the filename and not in the path.